src/pages/partners/affiliate-ib/_deriv-ib-programme.js

```diff
@@ -218,7 +218,12 @@ const DerivIBProgramme = () => {
                     </SubtitleHeader>
                 </TitleWrapper>
                 <IBSectionContainer padding="4rem 0 9.6rem 0">
-                    <StyledHeaderCommission as="h4" mb="1.6rem" size="1.6rem">
+                    <StyledHeaderCommission
+                        as="h4"
+                        type="sub-section-title"
+                        mb="1.6rem"
+                        size="1.6rem"
```

Comment:
Do we need `size` here, we set `font-size` in [`StyledHeaderCommission`](https://github.com/binary-com/deriv-com/blob/8c511ffdee92a19622e744b8dd291f1260803f35/src/pages/partners/affiliate-ib/_deriv-ib-programme.js#L134)